src/node_options.cc

```diff
@@ -615,7 +615,7 @@ PerIsolateOptionsParser::PerIsolateOptionsParser(
   Implies("--report-signal", "--report-on-signal");
 
   AddOption("--experimental-top-level-await",
-            "enable experimental support for ECMAScript Top-Level Await",
```

Comment:
we should support `--no-harmony-top-level-await`. Not entirely sure how it should be wired up.

Reply:
just tested locally and it "just works". Including that flag results in 
```
await new Promise((resolve) => {
^^^^^

SyntaxError: Unexpected reserved word
```

Reply:
right it already works with V8, but we need to not return the evaluation result if the flag is passed.

Reply:
Test case, if it helps:

```js
// Flags: --harmony-no-top-level-await

const common = require('../common');

process.on('unhandledRejection', common.mustCall());
Promise.reject(new Error('should not be fatal error'));
```

Reply:
That test is failing even without this change... not sure how to best test to make sure that `--no-harmony-top-level-await` is working properly but AFAICT it is wired up as expected now.

Reply:
that test should always pass, no matter what flags are passed.